convert2rhel/pkgmanager/handlers/yum/__init__.py

```diff
@@ -237,7 +238,7 @@ def _perform_operations(self):
                 diagnosis="Repository mirrors failed with error %s." % (str(e)),
             )
 
-    def _resolve_dependencies(self, validate_transaction):
+    def _resolve_dependencies(self):
```

Comment:
After removing the `validate_transaction` parameter there is a leftover in the comment section

Reply:
applied